RobotCode/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -12,20 +14,26 @@ public class RobotContainer
 
 	private final ShuffleboardTab ShuffleboardTab = Shuffleboard.getTab("Tab 1");
 
-	private final SendableChooser<Command> autoChooser = new SendableChooser<>();
+	private SendableChooser<Command> autoChooser = new SendableChooser<>();
 
 	/** The container for the robot. Contains subsystems, OI devices, and commands. */
 	public RobotContainer()
 	{
 		instance = this;
 
+		registerNamedCommands();
 		addAutonomousOptions();
 
 		// Configure the button bindings
 		configurePrimaryBindings();
 		configureSecondaryBindings();
 	}
 
+	private void registerNamedCommands()
+	{
+
```

Comment:
if there is supposed to be something here change it. otherwise is fine

Reply:
That method is intentionally empty. It'll be expanded later as we add commands.